pkgs/by-name/du/dummyhttp/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dummyhttp";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "svenstaro";
+    repo = pname;
```

Comment:
It is not recommended to reference `pname` from behind. ref: https://github.com/NixOS/nixpkgs/issues/277994